commands/build.go

```diff
@@ -737,8 +737,30 @@ func checkWarnedFlags(f *pflag.Flag) {
 	}
 }
 
-func writeMetadataFile(filename string, dt interface{}) error {
-	b, err := json.MarshalIndent(dt, "", "  ")
+func writeMetadataFile(filename string, response map[string]interface{}) error {
+	out := make(map[string]interface{})
+	for k, v := range response {
+		var dt []byte
+		j, ok := v.(string)
```

Comment:
If this means that any base64 string is decoded then I don't think that can be done as false positives can happen when regular string happens to be valid base64. Valid JSON array could work though, assuming that previous versions of buildx does not produce an error (previous versions failing to decode is unavoidable I guess).

Reply:
I'm not sure if this comment was addressed. Looks like any value can be base64 encoded now. To avoid false positives, should be limited to JSON arrays and objects. Eg. "MTU0" should not be decoded to "154"

Reply:
But it does change the valid values from JSON objects to any value afaics